cdk/lib/registration.ts

```diff
@@ -17,43 +14,8 @@ export class Registration extends GuStack {
 		// Until this project has been fully migrated to GuCDK you should update the 'old' infrastructure by modifying
 		// the YAML file and then re-running the snapshot tests to confirm that the changes are being pulled through by
 		// CDK
-		const cfnTemplate = new CfnInclude(this, 'YamlTemplate', {
+		new CfnInclude(this, 'YamlTemplate', {
 			templateFile: yamlTemplateFilePath,
 		});
-
-		const registrationLoadBalancer = cfnTemplate.getResource(
```

Comment:
This might look a little worrying, but the snapshots confirm that we're not actually deleting the load balancer 😅  

This is just a _reference_ to the [load balancer which is defined via YAML](https://github.com/guardian/mobile-n10n/blob/ff47b2b73d72bb99ec5935e02a8f372b4cdfb06c/cdk/lib/__snapshots__/registration.test.ts.snap#L1253-L1301).